components/switch/blender/recipe.js

```diff
@@ -0,0 +1,166 @@
+import { GEL } from '@westpac/core';
+import React from 'react';
+
+import { Switch } from '@westpac/switch';
+
+export function AllStyles({ brand }) {
+	return (
+		<GEL brand={brand}>
+			<Switch name="example-small" size="small" label="example" />
+			<Switch name="example-small" size="small" label="example" />
+			<Switch name="example-small-checked" size="small" label="example" checked />
+			<Switch name="example-small-block" size="small" label="example" block />
+			<Switch name="example-small-block-disabled" size="small" label="example" block disabled />
+			<Switch name="example-small-block-checked" size="small" label="example" block checked />
```

Comment:
What about `block checked disabled` combination?

Reply:
I already have it, its just further down in the examples